shared_model/backend/protobuf/proto_query_response_factory.hpp

```diff
@@ -6,6 +6,8 @@
 #ifndef IROHA_PROTO_QUERY_RESPONSE_FACTORY_HPP
 #define IROHA_PROTO_QUERY_RESPONSE_FACTORY_HPP
 
+#include "boost/none.hpp"
```

Comment:
No need to reference `none` in the header file.

Reply:
Even optional is not needed here. Removed both.